db/migrate/20230425193518_add_column_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddColumnToUsers < ActiveRecord::Migration[7.0]
```

Comment:
the name convention for this kind of migration tipically is `Add<Column Name>To<Table Name>`. In this case it should be `AddUsernameToUsers`